data/sql/updates/pending_db_world/razormawdamageupdate.sql

```diff
@@ -0,0 +1,3 @@
+-- damage modifier edit of Razormaw from 7.5 to 1.36 based on mangos values
+
+UPDATE `creature_template` SET `DamageModifier` = 1.36 WHERE `entry` = 17592 AND `modelid1` = 17256;
```

Comment:
Modelids are sniffed. Until probabilities get implemented we need to use scripting to change models instead, but this looks like leftover, though :p
```suggestion
UPDATE `creature_template` SET `DamageModifier` = 1.36 WHERE `entry` = 17592;
```

Reply:
The model isn't getting changed here though, just being used in tandem with the entry for the WHERE section.

Reply:
Oh right, I didn't notice it :c 
It's not needed in this case